README.md

```diff
@@ -374,6 +375,18 @@ $ git submodule update --init --recursive
 7. Add changelog to the GitHub release
 8. Finally: `npm publish`
 
+## Donate
```

Comment:
Update TOC (ideally by adding a TOC heading + re-enabling hallmark on the readme)